esp/scm/ws_workunits.ecm

```diff
@@ -898,6 +898,7 @@ ESPrequest WULogFileRequest
     [min_ver("1.38")] string LogDate;
     [min_ver("1.38")] int SlaveNumber(1);
     [min_ver("1.55")] int64 SizeLimit(0);
+    [min_ver("1.77")] string ErrorMessageOption;
```

Comment:
Maybe ErrorMessageFormat?

Reply:
Also should this be an enumeration?  If nothing else it makes it easier to get the value right.  Here they have to know the value they want is "text".